pkgs/desktops/gnome-3/extensions/tilingnome/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-tilingnome-${version}";
```

Comment:
```suggestion
  pname = "gnome-shell-extension-tilingnome";
```

Reply:
Are you sure? The manual only mentions `pname` in the section about Python packages.  Has it become the supported interface to `stdenv.mkDerivation` recently?

Reply:
`name = "name-${version}"` isn't needed anymore, but the manuals might not have been updated.

Reply:
cool, I guess I missed when that happened.